README.md

```diff
@@ -2,5 +2,62 @@
 
```

Comment:
The title should be updated as well to reflect the repository name change.

Reply:
done